clang/unittests/Format/TokenAnnotatorTest.cpp

```diff
@@ -2193,6 +2193,11 @@ TEST_F(TokenAnnotatorTest, UnderstandTableGenTokens) {
   ASSERT_TRUE(Keywords.isTableGenDefinition(*Tokens[0]));
   ASSERT_TRUE(Tokens[0]->is(Keywords.kw_def));
   ASSERT_TRUE(Tokens[1]->is(TT_StartOfName));
+
+  // Code, the multiline string token.
```

Comment:
Multiline string means it can span multiple lines? Maybe add a test for one that does?

Reply:
Added the multiline case.